pypeit/spectrographs/keck_hires.py

```diff
@@ -385,6 +413,97 @@ def check_frame_type(self, ftype, fitstbl, exprng=None):
         msgs.warn('Cannot determine if frames are of type {0}.'.format(ftype))
         return np.zeros(len(fitstbl), dtype=bool)
 
+    def vet_assigned_ftypes(self, type_bits, fitstbl):
+        """
+        Check the assigned frame types for consistency.
+        Args:
+            type_bits (`numpy.ndarray`_):
+                Array with the frame types assigned to each frame
+            fitstbl (`PypeItMetaData`_):
+                Table with the metadata for the frames to check.
+
+        Returns:
+            `numpy.ndarray`_: The updated frame types.
+
+        """
+        type_bits = super().vet_assigned_ftypes(type_bits, fitstbl)
+
+        # If both pixelflat and slitless_pixflat are assigned in the same configuration, remove pixelflat
+
+        # where slitless_pixflat is assigned
+        slitless_idx = fitstbl.type_bitmask.flagged(type_bits, flag='slitless_pixflat')
+        # where pixelflat is assigned
+        pixelflat_idx = fitstbl.type_bitmask.flagged(type_bits, flag='pixelflat')
```

Comment:
I added more documentation in the docstring to make it more clear. This is only checking that single frames aren't identified as both pixelflat and slitless_pixflat. Because of that, if a frames is associated with multiple configurations, the frametype will be modified for all configuration. So, no, it's not possible to slitless_pixflat type for one calibration group and pixelflat for another. I added that as well in the doctring.